### PR TITLE
tests: don't crash if no users exist when cleaning up login sessions

### PR DESCRIPTION
**Pull request by KKoukiou**
When no files exist in the directory the existing code would try to use the * as a file:
    
kkoukiou@o2:~/k$ for f in *; do echo $f; done
*

Instead let's list the files with `ls` to avoid this corner case as this breaks anaconda tests.

### CODE REVIEW

**Comment by KKoukiou** [comment]
See the failure at https://github.com/rhinstaller/anaconda-webui/pull/110

File: test/common/testlib.py [hunk 1]
```diff
@@ -1604,7 +1604,7 @@ def terminate_sessions():
 
             # Restart logind to mop up empty "closing" sessions, and clean user id cache for non-system users
             self.machine.execute("systemctl stop systemd-logind; cd /run/systemd/users/; "
-                                 "for f in *; do [ $f -le 500 ] || rm $f; done")
+                                 "for f in $(ls); do [ $f -le 500 ] || rm $f; done")
```
Comment (martinpitt) [inline test/common/testlib.py:1607]: FTR: It would be more efficient to do `[ -e $f -a $f -le 500 ]`, as that avoids spawning a shell and `ls`.

**Review by martinpitt — APPROVED** [review]
